sourced/ml/mining/models/stdlib.py

```diff
@@ -0,0 +1,63 @@
+from modelforge import merge_strings, Model, register_model, split_strings
+from sourced.ml.core.models.license import DEFAULT_LICENSE
+
+
+@register_model
+class StandardLibraries(Model):
+    """
+    List of standard library contents for each Babelfish language.
+    """
+
+    NAME = "stdlib"
+    VENDOR = "source{d}"
+    DESCRIPTION = (
+        "The list of standard library contents for C++, C#, Go, Java, JavaSript, PHP, "
+        "Python and Ruby. Language version is not taken into account, data was collected "
+        "as exhaustively as possible."
+    )
+    LICENSE = DEFAULT_LICENSE
+
+    def construct(self, libraries, infos):
+        self._libraries = libraries
```

Comment:
How about this if I got it right:
```suggestion
        self._library_names = libraries
```

Reply:
yeah it's more accurate, will replace here and everywhere